dynamicforms/mixins/password_field.py

```diff
@@ -0,0 +1,10 @@
+class PasswordFieldMixin(object):
+
+    def __init__(self, *args, password_field=False, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.password_field = password_field or False
+
+    def to_representation(self, value, row_data):
+        if self.password_field:
```

Comment:
Tole bi moralo biti v `__init__`. Tako kot je sedaj, uorabnik ne more povoziti s svojo varianto, ker vsakič on render to povoziš.